docs/dev/api/rdc-access-api/introduction.md

```diff
@@ -0,0 +1,41 @@
+---
+id: real-device-access-api-introduction
+title: Real Device Access API v2
+sidebar_label: Introduction
+---
+<p><span className="sauceGreen">Open Beta</span></p>
+
+:::info
+**This API is in Open Beta.**
+Access is available for customers with **Private Devices** only. The API specification is subject to change as we refine the product.
```

Comment:
we should make it clear that we don't plan on introducing breaking changes to the current API.